src/borg/archiver.py

```diff
@@ -1687,6 +1722,11 @@ def build_parser(self, prog=None):
         subparser.add_argument('location', metavar='TARGET', nargs='?', default='',
                                type=location_validator(),
                                help='archive or repository to delete')
+        group = subparser.add_mutually_exclusive_group()
+        group.add_argument('--oldest', dest='oldest', metavar='n', default=0, type=int,
+                           help='delete n oldest archives')
+        group.add_argument('--latest', dest='latest', metavar='n', default=0, type=int,
```

Comment:
as borg check already has --last, maybe use it here also. and --first. both for consistency.